wallets/provider-walletconnect-2/src/helpers.ts

```diff
@@ -192,3 +201,76 @@ export function getChainIdByNetworkName(
 
   return chainId;
 }
+
+export async function switchOrAddEvmChain(
```

Comment:
Please use `caip` package for working with CAIP standard. 
It can parse and also generate the string. Please make sure you are using it in this function.

Reply:
fixed